### ISSUE TITLE
cannot work with NVDA About dialog with screen review

### ISSUE THREAD

**Issue by nishimotz** [nvdajp/focusHighlight#17]
I received the report after releasing version 6.3.
There are similar reports with past versions.
The information from the reporter is cited below.

---

Go to the nvda help menu, about. Switch to screen review, and read up with 
numpad 7 several lines quickly. Maybe have to read back down with 9 a couple 
times, if you get to the top.

But somewhere in there, speech will stop getting interrupted. Once it finishes 
reading the line where that happens, NVDA freezes solid.

```
DEBUGWARNING - watchdog._watcher (03:26:24.651):
Trying to recover from freeze, core stack:
   File "nvda.pyw", line 231, in <module>
   File "core.pyc", line 519, in main
   File "wx\core.pyc", line 2134, in MainLoop
   File "wx\core.pyc", line 3240, in <lambda>
   File "gui\__init__.pyc", line 161, in showGui
   File "gui\__init__.pyc", line 542, in onActivate
   File "gui\__init__.pyc", line 303, in onAboutCommand
   File "gui\__init__.pyc", line 602, in messageBox
   File "gui\__init__.pyc", line 1003, in Notify
   File "core.pyc", line 490, in run
   File "queueHandler.pyc", line 86, in pumpAll
   File "queueHandler.pyc", line 53, in flushQueue
   File 
"C:\Users\myUser\AppData\Roaming\nvda\addons\focusHighlight\globalPlugins\focusHighlight.py", 
line 600, in update
   File 
"C:\Users\myUser\AppData\Roaming\nvda\addons\focusHighlight\globalPlugins\focusHighlight.py", 
line 483, in updateLocations
   File 
"C:\Users\myUser\AppData\Roaming\nvda\addons\focusHighlight\globalPlugins\focusHighlight.py", 
line 366, in updateNavigatorLocation
   File 
"C:\Users\myUser\AppData\Roaming\nvda\addons\focusHighlight\globalPlugins\focusHighlight.py", 
line 317, in moveAndShowWindow
```
